.github/dependabot.yml

```diff
@@ -4,4 +4,20 @@ updates:
     directory: /
     schedule:
       interval: weekly
-    rebase-strategy: disabled
\ No newline at end of file
+    rebase-strategy: disabled
```

Comment:
Sorry, not sure why it shows `dependabot.yml` in the diff. These changes were made in PR https://github.com/kubecost/cost-analyzer-helm-chart/pull/2545